raiden-ts/src/raiden.ts

```diff
@@ -1388,13 +1388,74 @@ export class Raiden {
     return receipt.transactionHash as Hash;
   }
 
+  /**
+   * Fetches our current UDC withdraw plan
+   *
+   * @returns Promise to object containing maximum 'amount' planned for withdraw and 'block' at
+   *    which withdraw will become available, and 'ready' after it can be withdrawn with
+   *    [[udcWithdraw]]; resolves to undefined if there's no current plan
+   */
+  public async getUdcWithdrawPlan(): Promise<
```

Comment:
We should use consistent spelling here.

We have `getUDCCapacity` and `depositToUDC` with all-caps UDC, and then `planUdcWithdraw` and `getUdcWithdrawPlan`.

Reply:
I agree, and maybe also either rename `depositToUdc` to `udcDeposit` or name `udcWithdraw` to `withdrawFromUdc`; we just need to be careful, since those are already part of the public API, that's why I just played along here. Let's discuss this.

Reply:
I like  `udcWithdraw` -> `withdrawFromUdc` more.

And I guess renaming it now before any stable release it better than doing it later.

Reply:
I totally agree with Paul. The `depositTo` and `withdrawFrom` sound better for function names. The others read more like getters or so. They do not include a verb.

Reply:
:heavy_check_mark: